### ISSUE TITLE
Points from Flat curve editor sometimes difficult to select if close to each other

### ISSUE THREAD

**Issue by Hombre57** [Beep6581/RawTherapee#4236]
The title says it all.

On the solution side : as suggested by @agriggio, we should be able to zoom to a certain part of the curve so we could act locally. Doable, but would like to have a broader opinion.

**Comment by Hombre57** [comment]
I'll probably enable zooming through the scroll wheel (like for the preview), while holding down the CTRL button (like for Combobox).

**Comment by agriggio** [comment]
that would be great!

**Comment by Hombre57** [comment]
@agriggio Just to avoid any confusion, the zoom will be horizontal only. I don't think it makes sense to zoom in both directions.

**Comment by agriggio** [comment]
@Hombre57 I agree about the proposed behaviour. However, I think we would also need some way of panning/sliding horizontally so that we can put the red hues towards the middle of the frame. Otherwise, editing those curves could still be difficult... thoughts?

**Comment by Beep6581** [comment]
Personally:
1. I have no issue with making fine adjustments using the tool the way it is. I hope the change doesn't complicate that.
2. I edit the red-yellow range most often, and the blues second-most often. However, the red-yellow range usually requires fine control, while the blue requires no finesse.
3. Being able to pan with wrap-around means you don't need to worry about which color to put on the edges to satisfy most users.

**Comment by agriggio** [comment]
I agree with 2 and 3, and I think the proposal by @Hombre57 won't affect 1 :-)
